test/suite/test_layered23.py

```diff
@@ -134,6 +137,27 @@ def update(self, table, count):
             self._append_single(table, entlist, k, v+k)
         return first_pos
 
+    # Remove some entries in the oplog for the table
+    def remove(self, table, count):
+        first_pos = len(self._entries)
+        entlist = self._get_entlist(table)
+
+        if len(entlist) == 0:
+            # If oplog has no entries for this table, silently succeed
+            return
+        incr = len(entlist)//count
+        if incr == 0:
+            incr = 1
```

Comment:
`incr` is not used.  Maybe to be used in the `range` statement following?

Reply:
I copied this from the update code. I'll remove this from both